src/axom/quest/tests/quest_sampling_shaper.cpp

```diff
@@ -844,6 +1038,477 @@ units: cm
   }
 }
 
+//-----------------------------------------------------------------------------
+
+TEST_F(SamplingShaperTest3D, basic_tet)
```

Comment:
The tests here and below add 3D testing for the sample-based shaper, including tests for the new projection lambdas